test/lib/ui5framework/AbstractResolver.js

```diff
@@ -15,7 +21,7 @@ test("AbstractResolver: abstract constructor should throw", async (t) => {
 	}, `Class 'AbstractResolver' is abstract`);
 });
 
-test("AbstractResolver: constructor", (t) => {
+test.serial("AbstractResolver: constructor", (t) => {
```

Comment:
Just fyi, for debugging/checking logs, there's also a `--serial` flag for AVA: https://github.com/SAP/ui5-project/blob/7c2033c1a5ab839d1aa3c7ebbd999e2b7cfbf3ef/package.json#L25